spec/services/profile_fields/remove_spec.rb

```diff
@@ -4,24 +4,19 @@
   context "when successfully removing a profile field" do
     it "removes the profile field and store accessor", :aggregate_failures do
       profile_field = create(:profile_field, label: "Removed field")
-      Profile.refresh_attributes!
       profile = create(:user).profile
 
-      expect(profile.respond_to?(:removed_field)).to be true
-      expect do
-        described_class.call(profile_field.id)
-      end.to change(ProfileField, :count).by(-1)
-      expect(profile.respond_to?(:removed_field)).to be false
+      expect { described_class.call(profile_field.id) }
+        .to change(ProfileField, :count).by(-1)
+        .and change { profile.respond_to?(:removed_field) }.from(true).to(false)
     end
 
     it "returns the correct response object", :aggregate_failures do
       profile_field = create(:profile_field, label: "Another Removed field")
-      Profile.refresh_attributes!
 
-      add_response = described_class.call(profile_field.id)
-      expect(add_response.success?).to be true
-      expect(add_response.profile_field).to be_an_instance_of(ProfileField)
-      expect(add_response.error_message).to be_blank
+      result = described_class.call(profile_field.id)
```

Comment:
Renamed this from `add_response` (clearly a copy/paste mistake) to `result` which makes more sense.